libs/utils/utils.js

```diff
@@ -840,7 +840,12 @@ async function loadPostLCP(config) {
 export function scrollToHashedElement(hash) {
   if (!hash) return;
   const elementId = hash.slice(1);
```

Comment:
Definitely an edge-case and non-standard, thus not requesting changes for it now, but maybe it will come up in the future: say an author adds a link in the form of `https://www.adobe.com/#foo?bar=baz`. The sliced hash would be considered to be `foo?bar=baz`, which might lead to unexpected behaviour.

Reply:
Maybe @suhjainadobe can look into this on top of writing some more unit tests (otherwise I could if once I have some more time on my hands...)

Reply:
sure, ill look into it today.